test/metabase/cmd/dump_to_h2_test.clj

```diff
@@ -26,26 +26,27 @@
 (deftest dump-deletes-target-db-files-tests
   ;; test fails when the application db is anything but H2 presently
   ;; TODO: make this test work with postgres / mysql / mariadb
-  (mt/with-temp-file [tmp-h2-db "mbtest_dump.h2"]
-    (let [tmp-h2-db-mv  (str tmp-h2-db ".mv.db")
-          file-contents {tmp-h2-db    "Not really an H2 DB"
-                         tmp-h2-db-mv "Not really another H2 DB"}]
+  (mt/with-temp-file [tmp-h2-db "mbtest_dump.h2"
```

Comment:
This test was failing because the destination h2 is not a valid h2 file.

Pretty sure we don't need the ability to dump to h2 on a random text file :), so I think it's okay to fix the test here.